docs/UserGuide.md

```diff
@@ -384,9 +385,53 @@ to `91234567` and `johndoe@example.com` respectively.
 
 [Back to Table of Contents](#table-of-contents)
 
-
 ### Managing Appointments
-[To be filled up]
+
+#### Adding Appointments : `addAppt`
+
+Allow doctors to schedule an appointment with a patient.
+
+Format: `addAppt APPOINTMENTNAME i/NRIC @d/DATE @t/TIME`
+
+{: .alert .alert-info}
+> :information_source: **Note:**
+>
+> * Adds Appointment Time and Appointment Name to the patient with the specified `NRIC` in MediBase3.
+> * Appointment time intervals cannot overlap with other existing appointment time intervals.
+> * You can schedule an appointment to a patient even if they are not being currently displayed in the Patient List Panel but doing so will refresh the panel to display all patients after their appointment has been added.
+> * Refer to the [Parameter Details](#parameter-details) section for more information on the purpose and constraints of each parameter.
+
+Example:
+* `addAppt Dental i/S1234567A @d/2024-10-27 @t/1100-1200` will schedule an appointment for the patient with NRIC `S1234567A` on `2024-10-27`, from `1100` to `1200` titled `Dental`. 
+
+{: .alert .alert-success}
+> :bulb: **Tip:**
+>
+> * Adding a new appointment will update the Appointment List Panel on the right with the new appointment details.
+> * Appointment List Panel is displayed in chronological order.
+
+[Back to Table of Contents](#table-of-contents)
+
+#### Deleting Appointments : `delAppt`
+
+Allow doctors to delete an appointment with a patient.
```

Comment:
Just a nit: I think it should not be "with a patient" as it sound like you are deleting the patient as well.

Reply:
Amended!